tests/idem/enum.rs

```diff
@@ -0,0 +1,31 @@
+// Enums test
+
+#[atrr]
+pub enum Test {
+    A,
+    B(u32, A /* comment */),
+    C,
+}
+
+pub enum Foo<'a, Y: Baz>
+    where X: Whatever
+{
+    A,
+}
+
+#[rustfmt_skip]
+enum Skip {}
+
+// C-style enum
+enum Bar {
+    A = 1,
+    B = 2, // comment
+    C,
+}
+
+enum LongVariants {
+    First(LOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOONG, // small comment
+          VARIANT),
+    // This is the second variant
+    Second,
+}
```

Comment:
Could you add an attribute or doc comment to one of the variants please? This is often a tricky edge case.